extras/log_file_client/lib/components/tank_card.dart

```diff
@@ -57,27 +57,27 @@ class TankCard extends StatelessWidget {
     );
   }
 
-  FutureBuilder<TankSnapshot> _graphThumbnailBuilder(
+  Expanded _graphThumbnailBuilder(
     Future<TankSnapshot> tankSnapshot,
```

Comment:
Rather than explicitly name the subclass that is returned, I typically just name `Widget` as the return type to simplify future refactorings.